habushu-mixology/pyproject.toml

```diff
@@ -7,7 +7,8 @@ license = "MIT License"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-krausening = "13"
+krausening = "16"
+cryptography = "^39.0.1"
```

Comment:
Q/A: Do we want to consider using a specific version of cryptography?

Reply:
We could - but my understanding is the due to transitive dependency challenges within PIP/Python in general, using ranges for libraries like this is preferrable

Reply:
Long but good read on taking this approach a step forward and **not** capping the upper bounds of dependency versions unless needed: https://iscinumpy.dev/post/bound-version-constraints/